Project.toml

```
@@ -1,6 +1,6 @@
 name = "ChainRules"
 uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "1.33.0"
+version = "1.33.1"
```

```suggestion
version = "1.34.0"
```

This is a new feature, not a bug fix